packages/react-flex/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@fluentui/react-flex",
+  "name": "@fluentui/react-flex-stale",
```

Comment:
q: why do we need to rename this package ?

Reply:
These changes were already part of the scaffolding of the package. Since I wanted to avoid breaking our existing `react-examples`, I moved the current implementation to a different package name. The plan is to then remove the implementation entirely at the end, before merging into master. Should I just delete it completely and start fresh?

Reply:
I suggest to remove it completely in a separate PR and scaffold a fresh package to avoid publishing of `@fluentui/react-flex-stale`. `@fluentui/react-flex-stale` will be published as it's not marked as a private and you will not be able to mark it as private as it's a dependency of `@fluentui/react-examples` 😯

Reply:
Would it not be published only after merging? This is strictly until I merge the converged component

Reply:
> Would it not be published only after merging?

Yep, only after a PR will be merged.

Reply:
So then it won't affect anything. I'll remove these from the current branch so I can exclusively merge the spec.